swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0):
```

Comment:
We should ideally remove `category` from the constructor arguments. We don't want the caller to be able to set a different category. While it's possible that someone could change the category on an instance by assigning directly to it after instantiation, not passing that to the constructor at least indicates that they shouldn't do that.